pkg/trace/agent/agent.go

```diff
@@ -273,6 +278,9 @@ func (a *Agent) Process(t pb.Trace) {
 	// Run both full trace sampling and transaction extraction in another goroutine.
 	go func(pt ProcessedTrace) {
 		defer watchdog.LogOnPanic()
+		defer func(start time.Time) {
+			metrics.Timing("datadog.trace_agent.debug.sample_ms", time.Since(start), nil, 1)
```

Comment:
Nit: this one isn't really only "sampling" maybe `process_trace_package_ms` would be more descriptive?

Reply:
This is the part where the trace gets sampled. If we were to factor this goroutine into a separate function, it would be called "sample"